### ISSUE TITLE
[Feature Request] LeechBlock integration

### ISSUE THREAD

**Issue by ashwinvis** [joelshepherd/tabliss#145]
I am now able to use TabBliss as a block screen with [LeechBlock](http://www.proginosko.com/leechblock/) by setting the `moz-extension://...` url in leechblock's settings for block page. LeechBlock uses this syntax

```sh
blocked.html?$S&$U
```

where `$U` is the url. Can tabliss accept inputs? See source: https://github.com/proginosko/LeechBlockNG/blob/master/blocked.html

**Comment by joelshepherd** [comment]
Hey! What would these inputs do?

**Comment by ashwinvis** [comment]
What I meant if tabliss can accept an input like `moz-extension://...?$S&$U`, it can display the url overriding / display alongside the quotes? I am not sure what $S is though. cc: @proginosko if he is willing to explain.

**Comment by joelshepherd** [comment]
Conceptually, I guess you could create a widget that looks for these inputs in the URL, and if they exist, display a message with them. It could hide itself the rest of the time.

**Comment by ashwinvis** [comment]
Yes that would work, but can it be done without modifications to the code? If so, please share a snippet.

**Comment by proginosko** [comment]
$S is the numerical identifier of the block set that triggered the block. It's passed to the blocking page (blocked.html) so that it can display the name of the block set (if a custom name has been specified) and the date/time when the blocked page will be available again. Hope this helps!

**Comment by ashwinvis** [comment]
A screenshot for reference:


![Screenshot_20190719_205224](https://user-images.githubusercontent.com/9155111/61558440-4fe0d980-aa67-11e9-8f32-be0098559090.png)

**Comment by ashwinvis** [comment]
Is it possible to add a Custom HTML widget to implement this as a user?
